libraries/websphere_profile.rb

```diff
@@ -47,17 +49,21 @@ class WebsphereProfile < WebsphereDmgr
         end
 
         # set java sdk if set
```

Comment:
could this comment be clarified/removed? I know it isn't part of your changes but it doesn't really make sense.